obelisk_ws/src/cpp/obelisk_control_cpp/src/example_position_setpoint_controller.cpp

```diff
@@ -0,0 +1,34 @@
+#include "rclcpp/rclcpp.hpp"
```

Comment:
can we have a `zoo` structure in the main `obelisk` package for this?

Reply:
done